db/migrate/20210714183848_change_imo_authorization_id_type_to_uuid.rb

```diff
@@ -0,0 +1,11 @@
+class ChangeImoAuthorizationIdTypeToUuid < ActiveRecord::Migration[5.2]
```

Comment:
I'm not sure this is really needed, but I noticed that I forgot to make this model use UUIDs.